go.mod

```diff
@@ -76,3 +78,5 @@ require (
 )
 
 go 1.16
+
+replace github.com/dhui/dktest => github.com/floatdrop/dktest v0.3.9-0.20211223122609-c60ae571a0e7
```

Comment:
This provides needed options (Mount and Hostname) for ydb docker image – should be removed, when https://github.com/dhui/dktest/pull/11/files is merged.